dicom_decompress/main.py

```diff
@@ -56,6 +56,8 @@ def decompress(dataset, ts):
         dataset['PixelData'].VR = 'OW'
     if dataset.pixel_array.size > 0:
         dataset.PixelData = dataset.pixel_array.tobytes()
+    if dataset.PhotometricInterpretation == 'YBR_FULL_422':
```

Comment:
I think we can make this more precise. Isn't the case we have evidence for where transfer syntax is jpeg baseline process 1 (1.2.840.10008.1.2.4.50) and photometric interpretation is YBR_FULL_42, so maybe we should restrict it to this for now?

Reply:
Good point, updated the code :)